### PR TITLE
Last 2 small fixes for pr2_moderation.php

### PR DESCRIPTION
**Pull request by bls1999**
Affected lines: 211-219

### CODE REVIEW

File: server/pr2_moderation.php [hunk 1]
```diff
@@ -209,14 +209,14 @@ function demote_moderator($socket, $name) {
 
 	if($from_player->group == 3){
 		$to_player = name_to_player($name);
+		if(isset($to_player) && $to_player->group < 2) {
+			$from_player->write('message`Error: '.$name.' is not a moderator.');
+		}
 		if(isset($to_player) && $to_player->group == 2) {
```
Comment (jacob-grahn) [inline server/pr2_moderation.php]: Should this be "else if"?

Comment (bls1999) [reply server/pr2_moderation.php]: It could be. I can change that

Comment (jacob-grahn) [reply server/pr2_moderation.php]: Oh, I see. Both statements can't be true at the same time. I'm just trying to figure out how this stops two messages from getting sent?

Comment (bls1999) [reply server/pr2_moderation.php]: If it's left as it is, this is the execution:

 - Demod
 - Check for a player's group and echo an error message if they're not a mod

Which produces two messages:
 - ____ has been demoted.
 - Error: ___ is not a moderator.

So, by changing the order we will prevent the double check and just go with one message.

Comment (jacob-grahn) [reply server/pr2_moderation.php]: Gotcha, thanks!

**Pull request merged**